servers/quarkus-common/src/main/java/org/projectnessie/quarkus/config/VersionStoreConfig.java

```diff
@@ -58,19 +58,6 @@ public boolean isNewStorage() {
   @WithDefault("INMEMORY")
   VersionStoreType getVersionStoreType();
 
-  /**
-   * Whether calls against the version-store are traced with OpenTracing/OpenTelemetry (Jaeger),
-   * enabled by default.
-   */
-  @WithName("trace.enable")
-  @WithDefault("true")
-  boolean isTracingEnabled();
```

Comment:
If I am following correctly, the idea is now to rely solely on which extensions are available at runtime, and let the annotations do their magic. Is that correct? One minor drawback of that is that if one extension is missing, traces or metrics won't be produced but we won't have any errors, not even warnings.

Reply:
Also, you probably need to remove the following lines as well:

https://github.com/projectnessie/nessie/blob/d17a9ed9e3ea29a8d8126f06886c1950527efbe2/events/quarkus/src/main/java/org/projectnessie/events/quarkus/config/QuarkusEventConfig.java#L39-L47

Reply:
Hm - metrics & tracing in for events already uses Quarkus stuff. I suspect, it's okay to leave it there.

Reply:
I think you should remove them, and replace the occurrences with `true`. It should be a really minor change.

Reply:
Yea, I tried, but there are indeed some conflicts. :(